scripts/deploy.sh

```diff
@@ -171,7 +171,7 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
 
   if [[ "${DEPLOY_TEMPEST}" == "yes" ]]; then
     # Deploy tempest
-    run_ansible os-tempest-install.yml
+    run_ansible ${BASE_DIR}/scripts/run_tempest.yml
```

Comment:
The ``tempest_install`` variable is already set to ``no`` in the group vars. Is that enough, or am I missing something?